lib/node/nodes/contour.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+var Node = require('../node');
+
+var TYPE = 'contour';
+
+var METHODS = ['nearest_neighbor', 'barymetric','IDW'];
+
+var CLASS_METHODS = ['equals', 'headstails', 'jenks', 'quantiles'];
+
+var PARAMS = {
+    // the scattered points dataset with real values
+    source: Node.PARAM.NODE(Node.GEOMETRY.POINT),
+    // column of values in source dataset
+    column: Node.PARAM.STRING(),
+    // buffer around my dataset
+    buffer: Node.PARAM.NULLABLE(Node.PARAM.NUMBER(), 0.2),
+    // 0: nearest neighbor, 1: barymetric, 2: IDW
+    method: Node.PARAM.NULLABLE(Node.PARAM.ENUM.apply(null,METHODS), METHODS[1]),
+    // classification method
+    class_method: Node.PARAM.NULLABLE(Node.PARAM.ENUM.apply(null,CLASS_METHODS), CLASS_METHODS[3]),
```

Comment:
It's a pretty clever trick ;) and any javascripter should understand it but it's a bit hard to read. The idea was:

```
var  NEAREST_NEIGHBOR_METHOD = 'nearest_neighbor_method';
...
    method: Node.PARAM.NULLABLE(Node.PARAM.ENUM(
        NEAREST_NEIGHBOR_METHOD,
        .....
    ), QUANTILES_METHOD),
...
```
but If you are OK using `.apply()`, I'm OK too.